vxfreeswitch/voice.py

```diff
@@ -92,25 +94,25 @@ def create_and_stream_text_as_speech(self, folder, command, ext, message):
         else:
             log.msg("Using cached voice file %r" % (filename,))
 
-        yield self.playback(filename)
+        yield self.output_stream(filename, settings)
 
     @inlineCallbacks
-    def send_text_as_speech(self, engine, voice, message):
+    def send_text_as_speech(self, engine, voice, message, settings={}):
         yield self.set("tts_engine=" + engine)
         yield self.set("tts_voice=" + voice)
-        yield self.execute("speak", message)
+        yield self.output_stream("say:'%s'" % message, settings)
```

Comment:
The docs are a bit confusing here. I have tried to use `speak` here, but it doesn't work. Using `say` here works exactly the same as `speak` in every test that I've tried. I got the idea from the example in this doc: https://freeswitch.org/confluence/display/FREESWITCH/mod_dptools:+play+and+get+digits

Even when I try to put a `say` command like `en name_spelled iterated ab733#`, it acts as if I'm passing a string to `speak`.